src/firecfg/firecfg.config

```diff
@@ -320,9 +320,9 @@ gcalccmd
 gcloud
 gconf-editor
 gdu
-geany
+#geany # text editor
```

Comment:
FTR: More IDE than just editor.

Reply:
> FTR: More IDE than just editor.

Sure, but it's probably lightweight enough to be used as the default text
editor.

Similarly to kate.